buildout.cfg

```diff
@@ -223,9 +224,12 @@ on_update = true
 shell = /bin/bash
 folders = mapproxy/ chsdi/*.py chsdi/lib/*.py chsdi/models chsdi/views chsdi/tests
 cmds =
-    FIXME=$(find ${validate-py:folders} -name \*.py | xargs ./buildout/bin/pep8 --ignore=E501)
+    FILES=$(find ${validate-py:folders} -name \*.py)
+    FIXME+=$(echo $FILES | xargs ./buildout/bin/pep8 --ignore=E501)
+    FIXME+=$(echo -e "\n ")
+    FIXME+=$(echo $FILES | xargs ./buildout/bin/pyflakes)
 
-    if [[ $FIXME == chsdi* || $FIXME == mapproxy* ]];
+    if [[ $FIXME == *"chsdi"* || $FIXME == *"mapproxy"* ]];
     then
       echo "$(tput setaf 1)$FIXME$(tput sgr0)" &&
       echo "You can fix automatically some styling errors using the following command:" &&
```

Comment:
Does pyflakes have an autocorrect mode as pep? The message here is pep specific only.